docs/src/callbacks.md

```diff
@@ -74,8 +74,9 @@ function SpeedyWeather.initialize!(
     callback.maximum_surface_wind_speed = zeros(progn.clock.n_timesteps + 1)
     
     # where surface (=lowermost model layer) u, v on the grid are stored
-    (; u_grid, v_grid) = diagn.layers[diagn.nlev].grid_variables
-    
+    u_grid = diagn.grid.u_grid[:,diagn.nlayers]
```

Comment:
Can you, in general, put a space ` ` after every `,` or `;` ?

Reply:
Just did that already in [4bab9bc](https://github.com/SpeedyWeather/SpeedyWeather.jl/pull/566/commits/4bab9bcdc31001777d09fe31f94f43f6f84102b7)